File: app/src/main/java/com/mrntlu/PassVault/Offline/Models/OthersObject.java
package com.mrntlu.PassVault.Offline.Models;

import io.realm.RealmObject;

public class OthersObject extends RealmObject {

    private String description;
    private String password;

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }
}

File: app/src/main/java/com/mrntlu/PassVault/Offline/Models/OthersObject.kt
package com.mrntlu.PassVault.Offline.Models;

import io.realm.kotlin.types.RealmObject;

class OthersObject(): RealmObject {
    var password: String = ""
    var description: String = ""

    constructor(password: String, description: String) : this() {
        this.password = password
        this.description = description
    }
}
